bump rails to 4.2.7

ulferts opened the pull request.

ulferts committed:
bump rails to 4.2.7

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://ulferts-fix-bump-rails-to-4-2-7.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://ulferts-fix-bump-rails-to-4-2-7.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Fix outdated count statement

This has been broken with 4.2.7:
https://github.com/rails/rails/blob/4-2-stable/activerecord/CHANGELOG.md#rails-427-july-12-2016
